packages/esm-appointments-app/src/calendar/header/calendar-header.component.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-import dayjs from 'dayjs';
+import { parseDate, toCalendar, createCalendar } from '@internationalized/date';
```

Comment:
Nit, but this should go, alphabetically, with the other imports that start with `@`